docs/software/uenv/guides.md

```diff
@@ -0,0 +1,237 @@
+[](){#ref-uenv-guides}
+# uenv guides
+
+This page contains guides for features and work
+
+[](){#ref-uenv-labels}
+## uenv labels
+
+Uenv are referred to using **labels**, which are used to refer to specific uenv, or groups of uenv, with the uenv command line tool.
+Each label has the following form:
+
+```
+name/version:tag@system%uarch
```

Comment:
in the table below you refer to it as `cluster`. I think that's the right term so the proposed change is

```suggestion
name/version:tag@cluster%uarch
```

Reply:
I see that you are using `system` also in ["deploying uenv"](https://docs.tds.cscs.ch/299/software/uenv/deploy/#versioning-and-labeling).

> uenv system[¶](https://docs.tds.cscs.ch/299/software/uenv/deploy/#uenv-system)
> The name of the [Alps cluster](https://docs.tds.cscs.ch/299/clusters/#alps-clusters) for which the uenv was built.

at this point I'm not sure anymore what's the right term 😅 

I'd just grep the folder and uniform the usage.

Reply:
So I fix any inconsistencies.